src/utils/DependencyUtils.ts

```diff
@@ -1,3 +1,21 @@
+import Game from "../model/game/Game";
+import ManifestV2 from "../model/ManifestV2";
+import ThunderstoreCombo from "../model/ThunderstoreCombo";
+import ThunderstoreVersion from "../model/ThunderstoreVersion";
+import { getCombosByDependencyStrings } from '../r2mm/manager/PackageDexieStore';
+
+/**
+ * ModPackDependencySetBuilderMode controls how dependencies are resolved for mod packs:
+ * - USE_EXACT_VERSION: Use exactly the version specified in the mod pack. (This is the default behaviour).
+ * - USE_LATEST_VERSION: Use the latest version instead
+ *
+ * For regular mods (non-modpacks), the latest version of dependencies is always used.
+ */
+export enum ModPackDependencySetBuilderMode {
```

Comment:
Would `ModpackDependencyStrategy` be a better name? "Builder" is an internal term of the helper function used by `getFullDependencyList`. It means or tells me nothing when used on `UpdateAllInstalledModsModal` call site. It's allso a bit shorter.